chain-tx-enclave-next/enclave-ra/ra-client/src/verifier.rs

```diff
@@ -212,19 +216,25 @@ impl EnclaveCertVerifier {
                 return Err(EnclaveCertVerifierError::MeasurementMismatch);
             }
 
-            if let Some(ref mr_enclave) = enclave_info.mr_enclave {
-                if mr_enclave != &quote.report_body.measurement.mr_enclave {
-                    return Err(EnclaveCertVerifierError::MeasurementMismatch);
+            match (
```

Comment:
would be good to add a comment about the validation logic here (https://github.com/crypto-com/chain-docs/blob/master/docs/modules/tdbe.md#svn-verification--compilation-order )

Reply:
https://github.com/crypto-com/chain/pull/1940#discussion_r454108375